### ISSUE TITLE
Github webhook to push to  local ReadTheDocs fails.

### ISSUE THREAD

**Issue by Lexrijvers** [rtfd/readthedocs.org#2352]
## Details

* Project URL: local
* Build URL (if applicable):
* Read the Docs username (if applicable): 

I want to set automatically push changes from our Github repo to our local ReadTheDocs. I can't find anything about what payload url you should use. So far, I'm using ```http://<domain>/api/v2/project/``` with content type application/json and without secret. 

## Expected Result

Changes are pushed to ReadTheDocs

## Actual Result

I receive a 403 error with {"detail":"Authentication credentials were not provided."}. I have no idea where to give credentials.

Basically I want to know how to setup a Github webhook for a local ReadTheDocs installation. Is anyone familiar with this?

**Comment by julyate** [comment]
I'm in the process of setting up a local ReadTheDocs instance. I can kick off my project builds by submitting a POST request to the following url:

`http://127.0.0.1:8000/build/my-docs/`

**Comment by Lexrijvers** [comment]
Great this worked!!

**Comment by julyate** [comment]
Great! If you're happy, you can go ahead and close the issue
